src/measures.jl

```diff
@@ -83,6 +83,23 @@ function measure(t::Tetrahedron)
   abs((A - D) ⋅ ((B - D) × (C - D))) / 6
 end
 
+"""
+    measure(splx::Simplex)
+
+Compute the measure (i.e. hyper-volume) for a simplex in any dimension using the Cayley-Menger Determinant.
+"""
```

Comment:
```suggestion
```

Reply:
I don't understand this diff, it's exactly the same (?).

Reply:
Why do you not want this docstring?

Reply:
The function is `measure` and it is already documented. We don't want to add redundant docstrings for all geometry types. The general docstring says: "Compute the measure of the geometry".

If you see the file, none of the other methods has a docstring, and that is intentional.